tests/domains/driving/test_driving.py

```diff
@@ -96,6 +107,8 @@ def test_elements_at(cached_maps):
             param crossing = network.crossingAt(spot)
             param intersection = network.intersectionAt(spot)
     """,
+        mode2D=use2DMap,
+        params={"use2DMap": use2DMap},
```

Comment:
Here and in the other tests, you shouldn't need to specify `use2DMap` since it will be turned on automatically in 2D mode and otherwise turned off.